dg/lib/openshift.py

```diff
@@ -102,44 +107,52 @@ def get_current_token() -> str:
         current OAuth access token stored in ~/.kube/config
     """
 
-    oc_cli_path = dg.config.data_gate_configuration_manager.get_oc_cli_path()
-    oc_whoami_command = [
-        oc_cli_path,
+    oc_whoami_args = [
         "whoami",
         "--show-token",
     ]
 
-    oc_whoami_command_result = subprocess.check_output(oc_whoami_command, text=True)
+    oc_whoami_command_result = execute_oc_command(oc_whoami_args)
 
-    return oc_whoami_command_result.rstrip()
+    return oc_whoami_command_result.stdout.rstrip()
 
 
-def get_oc_login_command_with_password(
+def get_oc_login_args_with_password(
     server: str, username: str, password: str
 ) -> List[str]:
-    oc_cli_path = dg.config.data_gate_configuration_manager.get_oc_cli_path()
-
-    return _get_oc_login_command_with_password(
-        str(oc_cli_path), server, username, password
-    )
-
+    return [
+        "login",
+        "--insecure-skip-tls-verify",
+        "--password",
+        password,
+        "--server",
+        server,
+        "--username",
+        username,
+    ]
 
-def get_oc_login_command_with_token(server: str, token: str) -> List[str]:
-    oc_cli_path = dg.config.data_gate_configuration_manager.get_oc_cli_path()
 
-    return _get_oc_login_command_with_token(str(oc_cli_path), server, token)
+def get_oc_login_args_with_token(server: str, token: str) -> List[str]:
+    return [
+        "login",
+        "--insecure-skip-tls-verify",
+        "--server",
+        server,
+        "--token",
+        token,
+    ]
 
 
 def get_oc_login_command_with_password_for_remote_host(
     server: str, username: str, password: str
 ) -> str:
     return " ".join(
-        _get_oc_login_command_with_password("oc ", server, username, password)
+        ["oc"] + get_oc_login_args_with_password(server, username, password)
```

Comment:
Don't you need the path of `oc`?

Reply:
No, this method returns a command to be executed on a remote host (i.e. an infrastructure node) where `oc` is contained in the search path.